docs/DeveloperGuide.adoc

```diff
@@ -295,59 +317,144 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use CardiBuddy
+
+|`* * *` |student |create new decks to contain my flashcards|so that I can organise my notes and modules
+
+|`* * *` |student |create test sessions|repeatedly test myself on the same content
+
+|`* * *`|student|test flashcards that I got wrong more often|better remember the content
+
+|`* * *`|student|immediately see the questions/flashcards that I get right|know what content I am more familiar with
+
+|`* * *` |student with many content-heavy modules|easily search for any flashcards that are relevant to my modules|more efficient with my time
+
+|`* * *`|student taking many modules|easily search for a deck that I want to access|more efficient with my time
+
+|`* * *` |student |delete decks of the modules that I am no longer taking|be more organised
+
+|`* * *` |student |edit my flashcards |change or add extra information whenever I want to
+
+|`* *`|student who receives flashcard images from her friends|drag and drop the images into the application|conveniently create new flashcards
+
+|`* *`|student who owns multiple mobile/computer devices|access flashcards on all my devices|revise them on the go
```

Comment:
I think this user story is quite similar to the one on line 354 (multi platform functionality), you might want to merge the two stories together.